res/database/Default/user/jorgaeff.xml

```diff
@@ -2020,9 +2020,9 @@
 			<!-- "Effekt: es stehen weniger Nachrichten zur Verfügung" -->
 		</news>
 
-		<news id="news-jorgaeff-lotto-01" type="0" creator="8936" created_by="jorgaeff">
+		<news id="news-jorgaeff-lotto-01" type="0" creator="8936" created_by="jorgaeff" comment="TODO: These never seem to appear. Check if changes to the availability script fixes this.">
```

Comment:
Maybe open a ticket (including the news ids to find them). Maybe these news should be handled like the stock news - initial trigger and then self-triggering at the right times.
Availability just restricts when the news can appear, so random selection of news has to hit them at the exact right time...

Reply:
#990 - the chance for these tickets to be randomly chosen is currently very low. The PR addresses this issue

Reply:
This almost certainly is a problem with scripttemplates too, the ones set with availability script appear very very rarely.

Reply:
I disagree. Currently, there are hardly any scripttemplates with availability script and in all instances the condition remains "true" whenever it is once met (e.g. daysplayed). So from that point on, it is like the availability is not restricted at all and the template is treated like every other one.

Reply:
I'm almost certain, how to make a test case?

Reply:
Locally delete the main scripttemplates file, kasi etc. leaving only a couple of db files containing templates. Then start a new game in dev-mode. Using the dev-screen (Tab) for the script agency, you can see the scripts currently available. The replaces offers action allows you ... well to do exactly that.
In the first days you should not see the templates with daysplayed>x. Fast forward to a later day (key 7 or 8) and replace the offers again.

Reply:
I know that, it's just more complicated because it seems it's a probability thing.